### PR TITLE
Fix several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7657,6 +7665,8 @@ cppp->cpp
 crace->grace, crate,
 craches->crashes, caches, crutches,
 craete->create
+craeted->created
+craetes->creates
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:7669]: Can you add `craetor` and `craetors`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -14864,6 +14881,9 @@ icesickle->icicle
 icluding->including
 iconclastic->iconoclastic
 iconifie->iconify
+icrease->increase
+icreased->increased
+icreases->increases
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:14886]: Add `icreasing`?
```suggestion
icrease->increase
icreased->increased
icreases->increases
icreasing->increasing
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -17192,6 +17212,7 @@ libreoficekit->libreofficekit
 libreries->libraries
 librery->library
 libries->libraries
+librray->library
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:17215]: Add `librraies`?
```suggestion
librraies->libraries
librray->library
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -22533,6 +22557,7 @@ rasons->reasons
 raspoberry->raspberry
 rathar->rather
 rathern->rather
+rationnal->rational, rationale,
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:22560]: Add `rationnals`?
```suggestion
rationnal->rational, rationale,
rationnals->rationals, rationales,
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -24172,6 +24197,9 @@ repulic->republic
 repulican->republican
 repulicans->republicans
 repulics->republics
+reputpose->repurpose
+reputposed->repurposed
+reputposes->repurposes
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:24202]: Add `reputposing`?
```suggestion
reputposes->repurposes
reputposing->repurposing
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -13,6 +13,10 @@ aadd->add
 aaggregation->aggregation
 aanother->another
 aapply->apply
+aaproximate->approximate
+aaproximated->approximated
+aaproximately->approximately
+aaproximates->approximates
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:19]: Add `aaproximating`?
```suggestion
aaproximates->approximates
aaproximating->approximating
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -7657,6 +7666,8 @@ cppp->cpp
 crace->grace, crate,
 craches->crashes, caches, crutches,
 craete->create
+craeted->created
+craetes->creates
 craeting->creating, crating,
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:7671]:
```suggestion
craetes->creates
craeting->creating, crating,
craetor->creator
craetors->creators
```

**Comment by bl-ue** [comment]
Btw, I wonder if you're interested in collaborating @luzpaz? Or maybe you just want to keep as you are?

**Review by bl-ue — APPROVED** [review]
Since this PR is large, I'll leave it open for a week for others' input, then I'll merge.

**Comment by luzpaz** [comment]
> Btw, I wonder if you're interested in collaborating @luzpaz? Or maybe you just want to keep as you are?

sure!

**Comment by bl-ue** [comment]
@luzpaz you've been [invited](https://github.com/orgs/codespell-project/teams/adm/discussions/3/comments/2) :D

**Comment by arm-in** [comment]
Only code coverage failed.
This seems not to make sense, since only the dictionary was modified.

**Comment by bl-ue** [comment]
Yeah, it sometimes does that.  However, it's an optional check, so the PR can still be merged.

**Comment by luzpaz** [comment]
Rebased and merged

**Pull request merged**